docroot/themes/contrib/civictheme/civictheme_library/components/00-base/typography/typography.stories.js

```diff
@@ -1,22 +1,8 @@
 // phpcs:ignoreFile
+import CivicThemeTypography from './typography.stories.twig';
+
 export default {
   title: 'Base/Typography',
 };
 
-export const Typography = () => {
```

Comment:
@govindmaloo 
Can you please re-instate this mechanism: it is used to auto-discover any existing or additional typography mappings.

Please use the generated HTML and pass it to twig in addition to the other HTML you have provided there